systemservices/marketplace/src/types/schema/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+        "service": {
+            "properties": {
+                "definition": {
+                    "properties": {
+                    },
+                    "type": "object"
+                },
+                "deployment": {
+                    "properties": {
+                        "source": {
+                            "type": "string"
+                        },
+                        "type": {
+                            "type": "string"
+                        }
+                    },
+                    "required": [
+                        "source",
+                        "type"
+                    ],
+                    "type": "object"
+                },
+                "hash": {
+                    "type": "string"
+                },
+                "hashVersion": {
+                    "type": "string"
+                },
+                "readme": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "definition",
+                "deployment",
+                "hash",
+                "hashVersion",
+                "readme"
```

Comment:
Readme could be empty.
Test if an empty string is ok or not.

Reply:
changing the type to something more accurate like this would probably solve this and add better auto completion
```ts
export interface Manifest {
  version: '1'
  service: {
    hash: string
    hashVersion: '1'
    deployment: {
      type: string
      source: 'ipfs'|'http'|'https'
    }
    definition: any,
    readme?: string
  }
}
```